DataContent/ReadingDB/ProcessorDataService.cs

```diff
@@ -11,81 +12,134 @@
+        //        }
+        //        _db.SaveChanges();
+        //    }
+        //}
+        //public void DeleteData(int id)
+        //{
+        //    using (_db = new ComputerContext())
+        //    {
+        //        var processor = _db.Processors.Where(p => p.Id == id).FirstOrDefault();
+        //        _db.Processors.Remove(processor);
+        //        _db.SaveChanges();
+        //    }
+        //}
+        //public Processor GetProcessor(string processorModel)
+        //{
+        //    //Deletes all trademarks and etc. Deletes sufix Processor
+        //    if (processorModel.Contains("Processor")) processorModel = processorModel.Substring(0, processorModel.IndexOf("Processor"));
+        //    processorModel = processorModel.DeleteSpecialChars();
+
+        //    using (_db = new ComputerContext())
+        //    {
+
+        //        //search for processor in DB
+        //        var processor = _db.Processors
+        //                        .Where(x => x.Model.Contains(processorModel)
+        //                        || processorModel.Contains(x.Model)).FirstOrDefault();
+
+        //        if (processor != null) return processor;
+        //        else
+        //        {
+        //            //try parse processor from Gpuskin database site
+        //            try
+        //            {
+        //                processor = new ProcessorParser().ParseProcessor(processorModel);
+        //                _db.Add(processor);
+        //                _db.SaveChanges();
+        //            }
+        //            catch (ProcessorNotFoundException ex)
+        //            {
+        //                //add new processor but throw exception, so it could be logged
+        //                processor = new Processor { Model = processorModel };
+        //                try
+        //                {
+        //                    processor.SetName(processorModel);
+        //                }
+        //                catch (ProcessorInvalidNameException e)
+        //                {
+        //                    ExceptionLogger.LogProcessorParsingException(processor, e);
+        //                }
+        //                using (_db = new ComputerContext())
+        //                {
+        //                    _db.Add(processor);
+        //                    _db.SaveChanges();
+        //                    processor = _db.Processors.Where(x => x.Model == processorModel).FirstOrDefault();
+        //                }
+        //                ExceptionLogger.LogProcessorParsingException(processor, ex);
+        //            }
+        //            return processor;
+        //        }
+        //    }
+        public void DeleteData(int id)
         {
-            using (_db = new ComputerContext())
-            {
-                var processors = _db.Processors.ToList();
-                return processors;
-            }
+            throw new NotImplementedException();
```

Comment:
will you change implementation of this method or it will always throw an Exception?

Reply:
My bad, I did not make this clear in my summary of the pull request.

The service classes are obsolete and will be deleted, these classes only stayed here to not spark any errors from the parse which still use them, when in fact they should be using the API as stated in the summary.